src/assets/specs/command-api.yaml

```diff
@@ -0,0 +1,108 @@
+openapi: "3.0.0"
+info:
+  title: "Rancher Desktop API"
+  version: 0.0.1
+paths:
+  /v0/list-settings:
+    get:
+      operationId: listSettings
+      summary:  List the current preference settings
+      responses:
+        '200':
+          description: |-
+            200 response
+          content:
+            application/json:
+              schema:
+                "$ref" : "#/components/schemas/preferences"
+
+  /v0/set:
+    put:
+      operationId: updateSettings
+      summary:  Updates the specified preference settings
+      requestBody:
+        description: |-
```

Comment:
`>-` is probably better here than `|-` (that is: the line breaks within the YAML file should not be forced).

Reply:
Sure.